sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/setup/SyncCreateAccountFragment.kt

```diff
@@ -91,14 +80,10 @@ class SyncCreateAccountFragment : DuckDuckGoFragment(R.layout.fragment_create_ac
     private fun renderViewState(viewState: ViewState) {
         when (viewState.viewMode) {
             is SignedIn -> {
-                binding.footerButton.isEnabled = true
-                binding.footerButton.setOnClickListener {
-                    listener?.launchFinishSetupFlow()
-                }
+                listener?.launchRecoveryCodeFlow()
             }
-
-            CreatingAccount -> {
-                binding.footerButton.isEnabled = false
+            else -> {
+                // nothing for now
```

Comment:
Can we then remove? I would like to avoid having `else` when using `when` + sealed class.